### PR TITLE
[MySQL] www, cdn, download, forge, wb

### PR DESCRIPTION
**Pull request by julianladisch**
add target www.mysql.com, handle some subdomains: cdn download forge wb, sort subdomains

### CODE REVIEW

**Comment by jsha** [comment]
From my machine, https://forge.mysql.com gives connection refused. Do you get a successful connection?

**Comment by julianladisch** [comment]
No, but http://forge.mysql.com/ redirects to http://www.mysql.com/, therefore I added the rule

    <rule from="^http://forge\.mysql\.com/"
            to="https://www.mysql.com/" />

File: src/chrome/content/rules/MySQL.xml [hunk 1]
```diff
@@ -59,6 +60,25 @@
...
 
+
+	<test url="http://cdn.mysql.com/Downloads/Connector-J/mysql-connector-java-5.1.34.tar.gz.md5" />
+	<test url="http://cdn.mysql.com/Downloads/MySQL-5.6/MySQL-5.6.23-1.linux_glibc2.5.i386.rpm-bundle.tar.md5" />
+
+	<!--  http://downloads.mysql.com/ -> https://dev.mysql.com/downloads/
+	      Other pages like http://downloads.mysql.com/archives/
+              don't have https pages
+	-->
+	<exclusion pattern="^http://downloads\.mysql\.com/." />
```
Comment (jsha) [inline src/chrome/content/rules/MySQL.xml]: This will fail the recently-added 'unescaped dot in exclusion' test in trivial-validate.py. Change it to \w?

**Comment by julianladisch** [comment]
I cannot find any 'unescaped dot in exclusion' test in https://github.com/EFForg/https-everywhere/blob/a0930a0220d6b8f289af012d234009b3b0e4ecf5/utils/trivial-validate.py

**Comment by jsha** [comment]
Sorry, I forgot my branch was still waiting on review:
https://github.com/EFForg/https-everywhere/pull/1129/files

**Pull request merged**